//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abcde_NO_VP.h
//
// Code generation for function 'abcde_NO_VP'
//

#ifndef ABCDE_NO_VP_H
#define ABCDE_NO_VP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abcde_NO_VP(double P_init, double V_init, double A_init, double P_wayp,
                 double V_wayp, double V_max, double A_min, double J_max,
                 double J_min, creal_T t[14]);

#endif
// End of code generation (abcde_NO_VP.h)
